client/app/components/Filters.jsx

```diff
@@ -102,6 +102,7 @@ function Filters({ filters, onChange }) {
                     allowClear={filter.multiple}
                     optionFilterProp="children"
                     showSearch
+                    maxTagCount={5}
```

Comment:
How about use 3 here to be consistent with parameters?

Reply:
Let's go with
```suggestion
                    maxTagCount={3}
                    maxTagTextLength={10}
                    maxTagPlaceholder={num => `+${num.length} more`}
```
then